airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/reports.py

```diff
@@ -4,11 +4,14 @@
 from __future__ import annotations
 
 import json
+import os
 import webbrowser
 from dataclasses import dataclass
-from typing import TYPE_CHECKING
+from types import MappingProxyType
+from typing import TYPE_CHECKING, Optional
+from zipfile import ZIP_DEFLATED, ZipFile
 
-from anyio import Path
+from pathlib import Path
```

Comment:
pathlib and anyio didn't play nice with each other. Since we were awaiting all the Path function, we might as well use the synchronous version from pathlib